app/views/sign_up_sheet/_table_header.html.erb

```diff
@@ -10,6 +10,7 @@
   <th width="10%">Bookmarks</th>
   <th width="10%">Actions</th>
   <th width="5%">Advertise- ment(s)</th>
+  <th width="5%">Link</th>
```

Comment:
Okay. I will try to do that.
I have handled the "30" size issue which you had earlier pointed out.
It is giving me some problem in CodeClimate which I am trying to figure out.